dsl/mixin/src/main/groovy/net/neoforged/gradle/dsl/mixin/extension/Mixin.groovy

```diff
@@ -0,0 +1,14 @@
+package net.neoforged.gradle.dsl.mixin.extension
+
+import groovy.transform.CompileStatic
+import net.minecraftforge.gdi.BaseDSLElement
+import net.minecraftforge.gdi.annotations.DSLProperty
+import org.gradle.api.provider.SetProperty
+
+@CompileStatic
+interface Mixin extends BaseDSLElement<Mixin> {
+    String EXTENSION_NAME = "mixin";
+
+    @DSLProperty
+    SetProperty<String> getConfigs();
```

Comment:
Should this not be a set of files?

Reply:
it actually can't be because it needs to be relative to the resources root